### DISCUSSION TITLE
[Errno 48] Address already in use

### DISCUSSION THREAD

**Discussion by sevoc** [Vector35/binaryninja-api#2962] [Q&A] [answered]
Hello! I have this error on log "[Errno 48] Address already in use" with the message "ERROR: Connection Refused" when I run the debugger locally. How to fix this? Thanks!

**Comment by xusheng6** [comment, marked as answer]
We are working on a C++ port of the debugger: https://github.com/Vector35/debugger_native. Currently the top priority is to wrap it up and push it to dev, then we can deal with various existing/new issues at https://github.com/Vector35/debugger_native/issues.
 
Please feel free to file an issue on the above repo. Please also try to provide more information, e.g., BN version, OS and version, etc, to help us reproduce it.
